api/server/router/image/image_routes.go

```diff
@@ -80,9 +82,12 @@ func (s *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrite
 		repo    = r.Form.Get("repo")
 		tag     = r.Form.Get("tag")
 		message = r.Form.Get("message")
+		q       = r.Form.Get("q")
```

Comment:
The `q` option should possibly be ignored on older API versions (although not a big issue)